test/jdk/java/awt/Mouse/MacTouchPressEventModifiers/MouseMacTouchPressEventModifiers.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
  * Copyright (c) 2022, JetBrains s.r.o.. All rights reserved.
  * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
```

Comment:
Not sure if we need to have Oracle and JetBrains copyright here?

Reply:
IIRC we need both Oracle and JetBrains copyrights here. 

You need to have two years in Oracle copyright: 1st - the year of the initial load; 2nd - the year of the last modification. In other words something like that:
`Copyright (c) 2022, 2023,`